### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by sul-devops-team**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -105,6 +105,9 @@ jobs:
       - node/install-packages:
           cache-key: yarn.lock
           pkg-manager: yarn
+      - run:
```
Comment (jcoyne) [inline .circleci/config.yml:108]: Is this necessary?

**Review by jcoyne — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (jmartin-sul) [reply .circleci/config.yml:108]: nope, it was a bit of flailing to reproduce a local build failure in CI to get another data point.  this is a pretty churny PR as-is, and i was planning to clean it up in a bit, but broke for dinner 🙂  (will put in draft in the meantime)

Comment (jmartin-sul) [reply .circleci/config.yml]: removed

**Review by jcoyne — APPROVED** [review]
Thank you.

**Pull request merged**